api/pkg/apis/projectcalico/v3/tier.go

```diff
@@ -49,6 +49,10 @@ type TierSpec struct {
 	// last.  Tiers with identical order will be applied in alphanumerical order based
 	// on the Tier "Name".
 	Order *float64 `json:"order,omitempty" protobuf:"bytes,1,opt,name=order"`
+	// DefaultAction specifies the action applied to workloads selected by a policy in the tier,
+	// but not rule matched the workload's traffic.
+	// [Default: Deny]
```

Comment:
```suggestion
	// [Default: Deny]
	// +kubebuilder:validation:Enum=Pass;Deny
```

Reply:
This ensures the underlying CRD enforces this as well (just in case someone edits that directly instead of going via our apiserver or calicoctl).

Reply:
Strictly speaking I think this should also be a pointer since it's an optional field